Automattic/wp-calypso
# Plan Overhaul revert

#### Proposed Changes

Addresses 779-gh-Automattic/martech
* Opens a branch to merge other Plan overhaul revert changes
* Disables the `plans/pro-plan` and `plans/starter-plan` flags for all environments



#### Testing Instructions

* Go to `/start/plans`
* You should see the Legacy plans (ie. Personal, Premium, Business, Ecommerce)
* Go to `/plans/[siteid]`
* You should see the Legacy plans (ie. Personal, Premium, Business, Ecommerce)


#### TODO
- [x] Confirm any `paid plans`, `Pro plan` or `Starter plan`  mentions are rolled back to the previous texts (ex. Business plan)
- [x] Update/roll back failing e2e tests
- [x] Remove legacy plan notice from the /plans page
